cmd/datamon/cmd/config.go

```diff
@@ -100,6 +105,17 @@ func (*CLIConfig) populateRemoteConfig(flags *flagsT) {
 	flags.context.Descriptor = contextDescriptor
 }
 
+func (c *CLIConfig) mustGetLogger(flags flagsT) *zap.Logger {
```

Comment:
cool 🎇 this sort of thing gives hope that there's a refactoring path to keep `cmd/` modular

* attach more dsl-ified funcs to types' namespaces of methods
* refactor types defining such dsl-ish things to a separate package

Reply:
> is mustGetLogger now used everywhere that a logger is initialized?
yes

> refactor types defining such dsl-ish things to a separate package
Yes. Indeed this is the goal I am pursuing. I'd like many little things like this (including the whole flagsT, with default settings and GCS strategy) to move to some `pkg/api/helpers` or `pkg/sdk/helpers` package to be available as an higher level API for instance, a python wrapper. Please suggest a name for that.

Reply:
follow-up note: comment only. No change requested